src-plugins/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -563,13 +657,35 @@ void itkFiltersToolBox::setupItkWindowingProcess()
 
 void itkFiltersToolBox::setupItkThresholdingProcess()
 {
-    d->process = dtkAbstractProcessFactory::instance()->createSmartPointer ( "itkThresholdingProcess" );
-    if (!d->process)
-        return;
-    d->process->setInput ( this->selectorToolBox()->data() );
-    d->process->setParameter ( d->thresholdFilterValue->value(), 0);
-    d->process->setParameter ( (double)d->thresholdFilterValue2->value(), 1);
-    d->process->setParameter ( (double)d->greaterButton->isChecked(), 2);
+    if ( d->binaryThreshold->isChecked() )
+    {
+        d->process = dtkAbstractProcessFactory::instance()->createSmartPointer( "itkBinaryThresholdingProcess" );
+        if (!d->process)
```

Comment:
you can remove this test, we should have this process, or see immediately if it's crashing

Reply:
This test is always in this class for each process of each sub-toolbox.
Are you sure we could remove it.
If yes, I will remove all of it in the class.

Reply:
Hum, since we started to remove it in other toolboxes, i think we can remove it here. Progressively we'll remove them :)